CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.15.3 (March 31, 2022)
+
+Fixed a bug in expiration_window behavior
+
+## 0.15.2 (March 21, 2022)
+
+Support for arm64 processors with Darwin OS - Publishment on Terraform Registry
```

Comment:
Let's just say "Resolved issue that prevented provider from being published to Terraform Registry"

Reply:
Got it